src/components/nav-bar/logo/Logo.js

```diff
@@ -9,7 +9,7 @@ const LogoHead = ({ mobile, color }) => {
     <React.Fragment>
       <Wrapper>
         <Link to="/">
-          <StyledLogo color={color} mobile={mobile} />
+          <StyledLogo color={color} mobile={mobile ? "mobile" : "desktop"} />
```

Comment:
at some point it's probably worth changing all of my dumbass boolean prop names to be prefixed with `is...` so `isMobile` to make it clearer what the prop is referring to

Reply:
Hahaha, was thinking that, but it's still quite readable, so might be renaming in future!